Helper/Data.php

```diff
@@ -211,19 +209,18 @@ public function collectModuleConfig(): array
             return $this->config[$storeId];
         }
 
-        try {
-            $this->config[$storeId]['enabled'] = $this->scopeConfig->getValue('sentry/environment/enabled', ScopeInterface::SCOPE_STORE)
-                ?? $this->deploymentConfig->get('sentry') !== null;
-        } catch (TableNotFoundException|FileSystemException|RuntimeException $e) {
```

Comment:
Does isSetFlag already handle these exceptions?
Could you test that out using the instructions here: https://github.com/justbetter/magento2-sentry/pull/125#issue-2065873558

So, setting a different nonexistent database connection and running magerun2 db:info